SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,793 @@
+            <div class="labels">
+                <span class="label special">SPECIAL</span>
+                <span class="label spicy">SPICY</span>
+            </div>
+            <img src="https://bootstrapmade.com/content/demo/NiceRestaurant/assets/img/restaurant/main-7.webp" alt="Signature Ribeye Steak">
+            <h2>Signature Ribeye Steak</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam accumsan risus ut dui pretium.</p>
+            <p class="price">$32.99</p>
+        </div>
+    </div>
+   
+   </Chef's-Specials>
```

HTMLHint (1.5.0)

[error] The html element name of [ Chef's-Specials ] must be in lowercase.

(tagname-lowercase)